source/sync/migrate-sync-modes.txt

```diff
@@ -343,28 +343,28 @@ generated subscriptions use a specific naming format: ``flx_migrated_`` + your
 object model's name. For example, a ``Person`` object model's subscription
 name would be ``flx_migrated_Person``.
 
-Refer to the SDK documentation for details about adding and removing
-subscriptions:
+You can keep the automatically-generated subscriptions if you don't want to
+remove and recreate them. But you should consider that because these 
```

Comment:
The "consider that because..." made this sentence read awkwardly to me on the first pass. maybe:
```suggestion
remove and recreate them. But because these subscriptions aren't created in your client code, they may make it harder for you to maintain and extend client code.
```